### PR TITLE
Updated samples, added call to ReleaseLicense

### PR DESCRIPTION
**Pull request by remusp**

### CODE REVIEW

File: LanguagePlatform/Sdl.SDK.LanguagePlatform.Samples.BatchExport/app.config
Comment (cornea-robert) [inline LanguagePlatform/Sdl.SDK.LanguagePlatform.Samples.BatchExport/app.config:1]: Please remove this

File: LanguagePlatform/Sdl.SDK.LanguagePlatform.Samples.BatchImport/app.config
Comment (cornea-robert) [inline LanguagePlatform/Sdl.SDK.LanguagePlatform.Samples.BatchImport/app.config:1]: Please remove this

File: LanguagePlatform/Sdl.SDK.LanguagePlatform.Samples.TmAutomation/ServerContainer.cs
Comment (cornea-robert) [inline LanguagePlatform/Sdl.SDK.LanguagePlatform.Samples.TmAutomation/ServerContainer.cs:1]: Please put back the examples

**Review by cornea-robert — CHANGES_REQUESTED** [review]

File: LanguagePlatform/Sdl.SDK.LanguagePlatform.Samples.BatchExport/app.config
Comment (remusp) [reply LanguagePlatform/Sdl.SDK.LanguagePlatform.Samples.BatchExport/app.config:1]: I added comments that explain why the added lines are needed.

File: LanguagePlatform/Sdl.SDK.LanguagePlatform.Samples.BatchImport/app.config
Comment (remusp) [reply LanguagePlatform/Sdl.SDK.LanguagePlatform.Samples.BatchImport/app.config:1]: I added comments that explain why the added lines are needed.

File: LanguagePlatform/Sdl.SDK.LanguagePlatform.Samples.TmAutomation/ServerContainer.cs
Comment (remusp) [reply LanguagePlatform/Sdl.SDK.LanguagePlatform.Samples.TmAutomation/ServerContainer.cs:1]: Done.

**Pull request merged**